packages/ember-model/tests/belongs_to_test.js

```diff
@@ -335,7 +335,7 @@ test("should be able to set nonembedded relationship to null", function() {
   });
 
   equal(post.get('author'), null);
-  deepEqual(post.toJSON(), {id: 1, author_id: null});
+  deepEqual(post.toJSON(), {id: 1, author_id: undefined});
```

Comment:
There seems to have been a change within Ember between `2.0` and `2.8`:

**2.0**:

<img width="166" alt="screen shot 2017-07-12 at 22 01 01" src="https://user-images.githubusercontent.com/2526/28139899-eddf37bc-674d-11e7-930d-b7245cc00c7c.png">

**2.8**:

<img width="167" alt="screen shot 2017-07-12 at 22 02 18" src="https://user-images.githubusercontent.com/2526/28139906-f557c4d2-674d-11e7-9cb4-dd0ba1a719ad.png">